python/paddle/v2/reader/decorator.py

```diff
@@ -323,3 +323,101 @@ def xreader():
                 yield sample
 
     return xreader
+
+
+def _buf2lines(buf, line_break="\n"):
```

Comment:
line break won't work in binary data, maybe we should let parser decide when to output a new data item?

Reply:
If `cut_lines=False` the binary data will send to parser directly. Do you mean by should let user's parser generate data, and make `pipe_reader` a decorator?

Reply:
Yes, I thought maybe pipe_reader should not cut the lines, since it does not have sufficient information, we might want leave it to the user's parser to do so (cut and generate data).

Reply:
Agree, will update.